### PR TITLE
SYNC-1607

### PR DESCRIPTION
**Pull request by shinnlok**

### CODE REVIEW

**Comment by Ithildir** [comment]
Hi! Should I review this pull or https://github.com/Ithildir/liferay-portal/pull/396?

**Comment by shinnlok** [comment]
This one. Thanks!

**Comment by shinnlok** [comment]
made a few minor changes, rebased, and force pushed. ci:retest

**Comment by shinnlok** [comment]
status? ci:retest

**Comment by Ithildir** [comment]
Reviewing it now!

File: modules/apps/sync/sync-api/bnd.bnd [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Bundle-Name: Liferay Sync API
 Bundle-SymbolicName: com.liferay.sync.api
 Bundle-Version: 1.0.2
-Export-Package: com.liferay.sync.constants
\ No newline at end of file
+Export-Package: com.liferay.sync.*
```
Comment (Ithildir) [inline modules/apps/sync/sync-api/bnd.bnd]: Do we really have to export everything? Also, please don't use wildcards but list explicitly what packages you want to export (SF enforces this).

File: modules/apps/sync/sync-service/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Bundle-Name: Liferay Sync Service
+Bundle-SymbolicName: com.liferay.sync.service
+Bundle-Version: 1.0.0
+Export-Package:\
+	com.liferay.petra.io.delta,\
```
Comment (Ithildir) [inline modules/apps/sync/sync-service/bnd.bnd]: Why this?

**Comment by Ithildir** [comment]
ci:stop

**Comment by shinnlok** [comment]
Force pushed removing wildcard.. without the io delta export:

org.osgi.framework.BundleException: Could not resolve module: com.liferay.sync.service [546]
  Unresolved requirement: Import-Package: com.liferay.petra.io.delta; version="[1.0.0,2.0.0)"

**Comment by Ithildir** [comment]
I think it's because `petra-io-delta` is not deployed, while we should. Trying...

**Comment by shinnlok** [comment]
Ah i see.

**Comment by Ithildir** [comment]
Sent to https://github.com/brianchandotcom/liferay-portal/pull/40244.

**Comment by shinnlok** [comment]
Thanks Andrea!

Just a reminder: https://github.com/Ithildir/liferay-portal-ee/pull/14 goes along with it.

**Pull request closed without merging**